public/app/features/scopes/selector/ScopesSelectorService.ts

```diff
@@ -255,7 +308,15 @@ export class ScopesSelectorService extends ScopesServiceBase<ScopesSelectorServi
 
   public closeAndApply = () => {
     this.updateState({ opened: false });
-    this.setNewScopes();
+    return this.setNewScopes();
```

Comment:
```suggestion
    return this.apply();
```
Any reason we don't want to do this?

Reply:
Yeah we probably can.